CoEpi/domain/model/KeyValueStore.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class KeyValueStore {
+    static let boolDefaults = ["showOnboarding": true]
```

Comment:
We don't need to cache the values. `UserDefaults` already does this.

Reply:
KeyValueStore no longer specifies initial values for each of the keys. Instead, if a key's value has not yet been set, then nil will be returned.